### ISSUE TITLE
[Bug] Won't click continue and leave groups on steam anymore.

### ISSUE THREAD

**Issue by JourneyOver** [Citrinate/gleamSolver#8]
Pretty much the title, seams like something has changed either on gleam.io or the script? that makes it so it can't click continue anymore for checking on steam group, it joins groups just fine, but it just gets stuck on actually verifying it on gleam and therfor it doesn't leave the group either..

```
VM324:399 Uncaught TypeError: entry.verifyEntryMethod is not a functionmarkEntryCompleted @ VM324:399(anonymous function) @ VM324:666(anonymous function) @ VM80:2f.string.d.(anonymous function) @ VM89:49
```

Hopefully that can maybe help out on figuring out what the issue is...

**Comment by JourneyOver** [comment]
Seems to have an issue with unfollowing twitter accounts as well, as I just noticed that I had follows still on some twitter accounts from some of the giveaways..

(and before you ask, yes I am running the instant-win Plus mode)

**Comment by Citrinate** [comment]
I've got the issue with Steam groups resolved on my end with v1.4.20.
